PowerShellTools.Test/IntelliSense/AutoCompletionControllerUnitTests.cs

```diff
@@ -26,7 +26,7 @@ public void ShouldCompleteBrace()
             _mockedScript = @"MyFunc";
             _mockedCaret = 6;
 
-            int actual = autoCompletionController.ProcessKeystroke((uint)VSConstants.VSStd2KCmdID.TYPECHAR, '(');
+            int actual = autoCompletionController.ProcessKeystroke(VSConstants.VSStd2KCmdID.TYPECHAR, '(');

```

Comment:
Do we have new unit tests to cover all these new scenarios and potential edge cases?